src/components/DataViewPrimaryFilter.jsx

```diff
@@ -75,8 +80,8 @@ function DataViewPrimaryFilter(props) {
 				type_${props.type}
 			`}
 		>
-			<BodyText className="filter">{props.label}:</BodyText>
-			<BodyText className="filter"><b>{props.value || "Any"}</b></BodyText>
+			<BodyText className="filter-label">{props.label}:</BodyText>
+			<BodyText className="filter-value"><b>{props.value || "Any"}</b></BodyText>
 			{
 				props.type === "optional" &&
 				<CloseIcon

```

Comment:
Please also on this branch change the variable valueString from "any" to "Any" on file "FilterDate.tsx"